src/boot/z_locale.c

```diff
@@ -6,10 +6,25 @@ s32 gCurrentRegion = 0;
 LocaleCartInfo sCartInfo;
 
 void Locale_Init(void) {
+#if PLATFORM_N64
+    ALIGNED(4) u8 regionInfo[4];
+    u8 countryCode;
+
+    osEPiReadIo(gCartHandle, 0x3C, (u32*)regionInfo);
+#else
+    u8 countryCode;
+
     osEPiReadIo(gCartHandle, 0x38, &sCartInfo.mediaFormat);
     osEPiReadIo(gCartHandle, 0x3C, &sCartInfo.regionInfo);
+#endif
 
-    switch (sCartInfo.countryCode) {
+#if PLATFORM_N64
+    countryCode = regionInfo[2];
+#else
+    countryCode = sCartInfo.countryCode;
+#endif
```

Comment:
why separate those into 2 `#if` blocks?

Reply:
huuuh... because that's how it came about and I didn't realize I could :smile: , I'll merge them